code/go/0chain.net/smartcontract/dbs/event/writemarker.go

```diff
@@ -49,7 +49,7 @@ type WriteMarker struct {
 //	if err := tx.Model(&Blobber{}).Clauses(clause.OnConflict{
 //		Columns:   []clause.Column{{Name: "blobber_id"}},
 //		DoUpdates: clause.Assignments(vs),
-//	}).Create(&Blobber{BlobberID: w.BlobberID, Used: w.Size, SavedData: w.Size}).Error; err != nil {
+//	}).Create(&Blobber{ID: w.ID, Used: w.Size, SavedData: w.Size}).Error; err != nil {
```

Comment:
Remove commented-out code, don't maintain it.

Reply:
we might need it in the future, let's wait until all snapshot aggregates are tested

Reply:
Leaving commented-out code is fine during development but should be removed before finally merging into the main code stream.